can/interfaces/usb2can/usb2canabstractionlayer.py

```diff
@@ -80,17 +80,25 @@ def __init__(self, dll="usb2can.dll"):
             log.warning('DLL failed to load at path: {}'.format(dll))
 
     def open(self, configuration, flags):
+        """
+        Opens a CAN connection using `CanalOpen()`.
+
+        :param bytes configuration: the configuration as ASCII bytes
+                                    (or simply as a str on Python 2)
+        :param int flags: the flags to be set
+
+        :raises can.CanError: if any error occured
+        :returns: Nothing
+        """
         try:
             # unicode is not supported
-            configuration = configuration.encode('ascii', 'ignore')
             result = self.__m_dllBasic.CanalOpen(configuration, flags)
             if result != CANAL_STATUS_OK:
```

Comment:
> I could not find any official mention of `CANAL_STATUS_OK`.

I just called it like that because I thought it was a simple name. ;)

> Any greater-than-zero return value should indicate success in this case.

OK.